market_maker_keeper/uniswapv2_util.py

```diff
@@ -0,0 +1,287 @@
+#
+# Copyright (C) 2020 MikeHathaway
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import pandas as pd
+from web3 import Web3
+from time import gmtime, strftime
+import calendar
+import json
+import ast
+import logging
+
+uniswap_abi = [{"name": "NewExchange", "inputs": [{"type": "address", "name": "token", "indexed": True}, {"type": "address", "name": "exchange", "indexed": True}], "anonymous": False, "type": "event"}]
```

Comment:
What do you think about breaking out the ABI like we do in pymaker into its own file?

Reply:
Yeah that's the approach taken elsewhere. This file was unneeded, so removed entirely.